<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1120</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1120</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1121</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.954273, y1:0.591, x2:0.352, y2:0.517, stop:0.323864 rgba(107, 28, 159, 255), stop:1 rgba(140, 28, 180, 255));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>26</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0.852273 rgba(255, 255, 255, 0), stop:0.875 rgba(255, 255, 255, 0), stop:0.886364 rgba(255, 255, 255, 0));</string>
     </property>
     <property name="text">
      <string>FIRSTsurvey</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>12</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>33</y>
       <width>12</width>
       <height>12</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>20</y>
       <width>10</width>
       <height>10</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0.852273 rgba(255, 255, 255, 0), stop:0.875 rgba(255, 255, 255, 0), stop:0.886364 rgba(255, 255, 255, 0));
color: rgb(255, 255, 255);
border-radius: 3px;</string>
     </property>
     <property name="text">
      <string>Sair</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	border: none;
	border-radius: 12px;
	background-color: rgb(140, 28, 180);
}
QPushButton:hover {
	background-color: rgb(107, 28, 159);
}</string>
    </property>
    <property name="text">
     <string>Dados do CPF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>1020</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 222, 255);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>488</y>
      <width>120</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(140, 28, 180);
	border: none;
	border-radius: 15px;
	background-color: rgb(255, 255, 255);
}
QPushButton:hover {
	color: rgb(107, 28, 159);
}</string>
    </property>
    <property name="text">
     <string>Consultar 
dívidas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>488</y>
      <width>120</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(140, 28, 180);
	border: none;
	border-radius: 15px;
	background-color: rgb(255, 255, 255);
}
QPushButton:hover {
	color: rgb(107, 28, 159);
}</string>
    </property>
    <property name="text">
     <string>Última
compra</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>490</y>
      <width>120</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(140, 28, 180);
	border: none;
	border-radius: 15px;
	background-color: rgb(255, 255, 255);
}
QPushButton:hover {
	color: rgb(107, 28, 159);
}</string>
    </property>
    <property name="text">
     <string>Consultas feitas
no CPF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>490</y>
      <width>120</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(140, 28, 180);
	border: none;
	border-radius: 15px;
	background-color: rgb(255, 255, 255);
}
QPushButton:hover {
	color: rgb(107, 28, 159);
}</string>
    </property>
    <property name="text">
     <string>Bens no
CPF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>475</y>
      <width>351</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(140, 28, 180);
border-radius: 15px;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>                  BEM VINDO!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0.852273 rgba(255, 255, 255, 0), stop:0.875 rgba(255, 255, 255, 0), stop:0.886364 rgba(255, 255, 255, 0));
color: rgb(55, 62, 85);</string>
    </property>
    <property name="text">
     <string>Firstsurvey ajudando a ter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0.852273 rgba(255, 255, 255, 0), stop:0.875 rgba(255, 255, 255, 0), stop:0.886364 rgba(255, 255, 255, 0));
color: rgb(55, 62, 85);</string>
    </property>
    <property name="text">
     <string>tranquilidade na vida</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0.852273 rgba(255, 255, 255, 0), stop:0.875 rgba(255, 255, 255, 0), stop:0.886364 rgba(255, 255, 255, 0));
color: rgb(55, 62, 85);</string>
    </property>
    <property name="text">
     <string>financeira</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>471</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Consulte o CPF, monitore seus dados de dívidas e últimos acessos.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>116</y>
      <width>491</width>
      <height>299</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/newPrefix/13.png);
border-radius: 20px;
background-color: rgb(140, 28, 180);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>116</y>
      <width>47</width>
      <height>299</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(140, 28, 180);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_12</zorder>
   <zorder>frame</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_5</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
